docs/development/bump-k8s.md

```
@@ -0,0 +1,49 @@
+     k8s.io/apimachinery@v0.<minor>.<patch> \
+     k8s.io/client-go@v0.<minor>.<patch> \
+     k8s.io/cloud-provider@v0.<minor>.<patch> \
+     k8s.io/component-base@v0.<minor>.<patch> \
+     k8s.io/apiserver@v0.<minor>.<patch> \
+     k8s.io/component-helpers@v0.<minor>.<patch> \
+     k8s.io/controller-manager@v0.<minor>.<patch> \
+     k8s.io/kms@v0.<minor>.<patch>
+   ```
+
+3. **Tidy modules**:
+   ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)